example/classes.zig

```diff
@@ -73,6 +73,11 @@ pub const Dog = py.subclass("Dog", &.{Animal}, struct {
         return pyrepr.appendSlice(")");
     }
 
+    pub fn __add__(self: *const Self, other: py.PyString) !*Self {
```

Comment:
I've also tested this with accepting other as Self and it works!

Reply:
Maybe add something in the docs saying that this will raise TypeError if other is not a string